www/grpc/wallet.go

```diff
@@ -29,10 +29,10 @@ func (*walletServer) mapHistoryInfo(hi []wallet.HistoryInfo) []*pactus.HistoryIn
 	for _, hi := range hi {
 		historyInfo = append(historyInfo, &pactus.HistoryInfo{
 			TransactionId: hi.TxID,
-			Time:          uint32(hi.Time.Unix()),
-			PayloadType:   hi.PayloadType,
-			Description:   hi.Desc,
-			Amount:        hi.Amount.ToNanoPAC(),
+			// Time:          uint32(hi.Time.Unix()),  // TODO: Fix me
```

Comment:
What is it issue this ToDo? please make issue if is a issue.

Reply:
It panics, we don't have time of committing the transaction yet